auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/EditAccessGroupsController.kt

```diff
@@ -26,8 +29,9 @@ class EditAccessGroupsController(
 
     private val logger = LoggerFactory.getLogger(javaClass)
 
-    // This endpoint takes a single user cn, single access group cn and a list of organisation codes.
+    // This endpoint takes a single user cn, single access group cn, a list of organisation codes and a comment.
     // It will assign the user to that access group and the given list of organisations for that access group.
+    // If a comment is posted, it also checks the comments for any modifications and updates them accordingly
```

Comment:
It doesn't "check the comment for any modifications". It takes the comment field and if present, appends it to the existing comment field.

Reply:
if (newComment.isNotEmpty() && newComment != currentUser.comment) - I'd have thought this is technically comparing the two?

Reply:
Yeah but I'm not sure `newComment != currentUser.comment` will ever really be relevant since the comment gets appended and doesn't replace the old comment. Even if the comment has only ever been set via bulk upload and you upload the same file twice, it's actually still going to update the comment.

Say the comment is "foobar" in the spreadsheet. First upload would set comment to "\nfoobar" and then second upload would set it to "\nfoobar\nfoobar". But I don't think this is worth worrying about or even a bad thing necessarily. It's fine to keep appending to the comment so they can see that the user was updated without digging through the audit trail.

Practically speaking, we're just always appending the comment and I think that's fine. That's what the ticket asked for.